packages/settings-library/tests/test_utils_logging.py

```diff
@@ -40,14 +40,10 @@ def _v(cls, value) -> str:
     assert settings.LOG_LEVEL == "DEBUG"
 
     assert (
-        settings.json()
-        == '{"SC_BOOT_MODE": null, "LOG_LEVEL": "DEBUG", "APPNAME_DEBUG": false}'
+        settings.model_dump_json()
+        == '{"SC_BOOT_MODE":null,"LOG_LEVEL":"DEBUG","APPNAME_DEBUG":false}'
     )
 
     # test cached-property
     assert settings.log_level == logging.DEBUG
-    # log_level is cached-property (notice that is lower-case!), and gets added after first use
```

Comment:
In Pydantic v2, it is possible to annotate `@computed_fields`. It must be specified in advance if we want to include it in the model serialization. By default `@computed_fields(repr=True)` for public fields, `@computed_fields(repr=False)` for private ones.

See: https://docs.pydantic.dev/2.0/usage/computed_fields/